nevergrad/functions/games/test_game.py

```diff
@@ -0,0 +1,16 @@
+from . import game
+import numpy as np
+from typing import Any
+from typing import List
+
+np.random.seed()
```

Comment:
tests must **always** be in a function (starting by `test_`). This is a script and can make a huge mess in tests

Reply:
I'll update it now